config/default.yml

```
@@ -627,6 +627,7 @@ Rails/RedundantForeignKey:
 Rails/RedundantPresenceValidationOnBelongsTo:
   Description: 'Checks for redundant presence validation on belongs_to association.'
   Enabled: pending
+  SafeAutoCorrect: false
   VersionAdded: '2.13'
```

Can you insert `VersionChanged: '<<next>>'`?

```suggestion
  VersionAdded: '2.13'
  VersionChanged: '<<next>>'
```

---

@koic made the change and squashed into existing commit